home/ivp-planning/advanced-tools/creating-screenshots.md

```diff
@@ -1,14 +1,36 @@
 # Creating Screenshots
+  
+It is possible to save the current content of the 3D- or 2D-Panel as a screenshot. To do so, first click the Screenshot-Button in either panel.  
+A new window will open, in which you will be prompted to name your screenshot.
 
-At any time it is possible to save the current content of the [3D-panel](../user-interface/the-3d-panel.md) as a screenshot. To do so, first click on the "Screenshot" icon in the toolbar of the 3D-panel.
+![](../../../.gitbook/assets/screenshot-creation.png)  
 
-![](../../../.gitbook/assets/iVP\_guide\_creating\_screenshots\_icon.jpg)
+## Changing the Screenshot Resolution
 
-A new window will open that lets you specify the resolution of the screenshot. If the "Open" option was checked, the screenshot will be opened in the default image viewer after it has been created.
+Before saving your screenshot, you have the option to change your screenshots resolution. Options range from 1280x720 to 7680x4320.
```

Comment:
We need to rephrase this to not include the image width here, as it depends on the aspect ratio.